app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\AdminLoginRequest;
+use App\Models\Admin;
+use App\Traits\LoginTrait;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Route;
+use Laravel\Passport\Client;
+
+class LoginController extends Controller
+{
+    use LoginTrait;
+
+    function adminRegister(Request $request)
+    {   
+    /**
+     * Get a validator for an incoming registration request.
+     *
+     * @param  array  $request
+     * @return \Illuminate\Contracts\Validation\Validator
+     */
+    $valid = validator($request->only('email', 'name', 'password','phone'), [
+        'name' => 'required|string|max:100',
+        'email' => 'required|string|email|max:255|unique:admins,email',
+        'password' => 'required|min:6',
+        'phone' => 'required|unique:admins,phone',
+    ]);
```

Comment:
Use Laravel FormRequest class to validate user's input
https://laravel.com/docs/7.x/validation#creating-form-requests